mappings/net/minecraft/structure/BastionGenerator.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_5179 net/minecraft/structure/BastionGenerator
```

Comment:
The structure is called "bastion remnant", so `BastionRemnantGenerator`

Reply:
Yes `BastionRemnantGenerator` rename please